diff --git a/node_modules/skyhelper-networth/types/ProfileNetworthCalculator.d.ts b/node_modules/skyhelper-networth/types/ProfileNetworthCalculator.d.ts
index aa6dfab..d3c526d 100644
--- a/node_modules/skyhelper-networth/types/ProfileNetworthCalculator.d.ts
+++ b/node_modules/skyhelper-networth/types/ProfileNetworthCalculator.d.ts
@@ -1,4 +1,4 @@
-import { Item } from './SkyBlockItemNetworthCalculator';
+import { Item } from './global';
 import { NetworthOptions } from './global';
 
 interface NetworthResult {
